### PR TITLE
hpcgap: sync lib/package.gi, partially sync lib/helpbase.gi

### PR DESCRIPTION
**Pull request by fingolfin**

### CODE REVIEW

File: lib/package.gi [hunk 1]
```diff
@@ -261,7 +269,7 @@ InstallGlobalFunction( InitializePackagesInfoRecords, function( arg )
                    and GAPInfo.PackagesRestrictions.( pkgname ).OnInitialization(
                            record ) = false then
                   Add( GAPInfo.PackagesInfoRefuseLoad, record );
-                elif LowercaseString( pkgname ) in ignore then
+                elif pkgname in ignore then
```
Comment (alex-konovalov) [inline lib/package.gi]: This is interesting - why to drop `LowercaseString`?

**Comment by fingolfin** [comment]
I think I fixed the build error, but got carried away and also made several other tweaks... including work on `README.hpcgap.md`, which is not quite done yet.

I'll sort this out tomorrow, and possibly split this into multiple PRs. Until I mark this as ready again, feel free to ignore it. Sorry for the noise.

**Comment by fingolfin** [comment]
Actually, sorted it out already now, so this can be reviewed normally now.

**Comment by fingolfin** [comment]
Try changing to `static inline ` ?

**Pull request merged**